packages/preview/navigation-logger/0.1.1/typst.toml

```diff
@@ -0,0 +1,27 @@
+# for a description of available keys, see https://github.com/typst/packages/?tab=readme-ov-file#package-format
+
+[package]
+name = "navigation-logger"
+version = "0.1.1"
+entrypoint = "src/lib.typ"
+authors = ["TristanDuncombe"]
+license = "Unlicense"
+description = "A Navigation Log, developed to replace multiple pages of notes and provide a succinct template to manage all the information needed as a Cross Country Pilot!"
+repository = "https://github.com/tristanduncombe/navigation-logger"
+keywords = ["navigation", "aviation", "flying", "log"]
+categories = ["utility"]
+exclude = [
+  ".github",
+  "docs",
+  "scripts",
+  "tests",
+  ".typstignore",
+  "Justfile",
+  "thumbnail-dark.svg",
+  "thumbnail-light.svg",
+]
+
+[template]
+path = "src"
```

Comment:
Your template should be in a dedicated subdirectory. The contents of this directory will be copied when creating a new project with your template. The Typst files in the template should import the package and provide some default contents.